README.md

```diff
@@ -42,10 +42,10 @@ Using HyperSwitch, you can:
 - **Improve success rates** with seamless failover and auto-retries
 - **Reduce processing fees** with smart routing
 - **Customize payment flows** with full visibility and control
-- **Increase business reach** with local / alternate payment methods
+- **Increase business reach** with local/alternate payment methods
 
 > HyperSwitch is **wire-compatible** with top processors like Stripe, making it
-> easy to integrate.
+> Easy to integrate.
```

Comment:
Nit: this need not be capitalized?

Reply:
Hey @goodolLakshay, I've updated the capitalization on the README. Previously, the rendered README looked like this, the 'E' being uppercase was clearly not preferable.

![Screenshot of the rendered README](https://github.com/juspay/hyperswitch/assets/22217505/98355b01-a41a-4a27-be8d-0a7fca44478b)